core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -635,10 +635,10 @@ class Executor[S <: Scheduling](val platforms: Seq[ExecutionPlatform],
   def openStreams(executionId: String): fs2.Stream[IO, Byte] =
     ExecutionStreams.getStreams(executionId, queries, xa)
 
-  def relaunch(jobs: Set[String])(implicit user: User): Unit = {
+  private def relaunchExecs(idList: Set[String], cond: (Set[String], Execution[S]) => Boolean)(implicit user: User): Unit = {
```

Comment:
I suggest keeping jobs as first argument and using predicate (or filter) as second

Reply:
or better, write a generic `private def relaunchExecutions(predicate: Execution[S] => Boolean)`

Reply:
done thx